docs/error-investigation-guide.md

```diff
@@ -0,0 +1,92 @@
+# Azure Error Investigation Guide
+
+- [Azure Error Investigation Guide](#azure-error-investigation-guide)
+    - [Overview](#overview)
+    - [Azure Data Factory Studio](#azure-data-factory-studio)
+      - [Getting to the Studio](#getting-to-the-adf-studio)
+      - [Monitoring your pipeline runs and diagnosing issues](#monitoring-your-pipeline-runs-and-diagnosing-issues)
+      - [Troubleshooting pipeline configuration issues](#troubleshooting-pipeline-configuration-issues)
+    - [Observing Pipeline Failure Trends using the Dashboard](#observing-pipeline-failure-trends-using-the-dashboard)
+
+## Overview
+This guide provides an overview of basic pipeline failure troubleshooting in Microsoft Azure. It assumes a user
+has an account with full access to their Azure environment and basic familiarity with some Azure concepts. Full
+Azure docs can be found [here.](https://learn.microsoft.com/en-us/azure/?product=popular)
+
+## Azure Data Factory Studio
+Our primary window into what's happening in our pipelines is the Azure Data Factory Studio. It provides a GUI
```

Comment:
Can we hyperlink Azure Data Factory Studio here?

Reply:
Unfortunately no, since the URL contains specific stuff like subscription ID etc, so their link is different than ours =(